py/private/py_binary.bzl

```diff
@@ -172,7 +173,18 @@ python = use_extension("@rules_python//python/extensions:python.bzl", "python")
 python.toolchain(python_version = "3.8.12", is_default = False)
 python.toolchain(python_version = "3.9", is_default = True)
 ```
-"""
+""",
+    ),
+    "package_collision": attr.string(
+        doc = """The action that should be taken when a symlink collision is encounted when creating the venv.
+A collision can occour when multiple packages providing the same file are installed into the venv. The possible values are:
+
+* "error": (default) When conflicting symlinks are found, an error is reported and venv creation halts.
+* "warning": When conflicting symlinks are found, an warning is reported, however venv creation continues.
+* "ignore": When conflicting symlinks are found, no message is reported and venv creation continues.
+        """,
+        default = "error",
```

Comment:
The current default is to error, but perhaps it should be warning?

Reply:
if the error message tells you how to suppress, that's most of the DX right there. It's nice to force users to acknowledge they're aware of the collision, so if it actually breaks their program they are more likely to remember a plausible cause.